frontend/components/Accounts/Forms/ContactInput.tsx

```diff
@@ -208,7 +249,21 @@ export const EditInput = ({ onConfirm, value, onChange }) => (
     </Flex>
 );
 
-const ContactInput = ({ route, addText, initialData, contactType }) => {
+// TODO: fix up layout so it's clearer what's primary, verified, unverified
```

Comment:
What's this TODO?

Reply:
this one is just layout - want to both line things up, make it obvious which is primary, and make it clearer / more alerting when one is unverified